packages/http-cors/README.md

```diff
@@ -49,6 +49,28 @@ npm install --save @middy/http-cors
  - `headers` (string) (optional): value to put in Access-Control-Allow-Headers (default: `null`)
  - `credentials` (bool) (optional): if true, sets the `Access-Control-Allow-Origin` as request header `Origin`, if present (default `false`)
 
+NOTES:
+- If another middleware does not handle and swallow errors, then it will bubble all the way up 
+and terminate the Lambda invocation with an error. In this case API Gateway would return a default 502 response, and the CORS headers would be lost. To prevent this, you should use the `httpErrorHandler` middleware before the `cors` middleware like this:
+
+```javascript
+const middy = require('middy')
+const { httpErrorHandler, cors } = require('middy/middlewares')
```

Comment:
Since this is a branch for `v1`, the way you import middlewares is different 


```js
const cors = require('@middy/http-cors')
```

Reply:
good catch, now updated 👍